templeton/src/main/resources/templates/questions/AxImagery.html

```diff
@@ -19,21 +19,20 @@
 
                     <h1>Use Your Imagination</h1>
 
-                    <p>Please think of an UPCOMING situation that you expect will make you feel anxious or scared.<p/>
-                    <p>Pick a situation that you worry might go badly, or that you expect will evoke symptoms related to anxiety (e.g., racing heart, sweating).<p/>
-                    <p>Any situation that will make you feel anxious or scared will work.<p/>
-                    <p>If no specific situation is upcoming, then just select a situation that you expect would make you feel anxious or scared IF you were to be that situation.<p/>
+                    <p>Pick a situation that you worry might go badly, or that you expect will evoke symptoms related to anxiety (e.g., racing heart, sweating).</p>
+                    <p>Any situation that will make you feel anxious or scared will work.</p>
+                    <p>If no specific situation is upcoming, then just select a situation that you expect would make you feel anxious or scared IF you were to be that situation.</p>
 
                     <label class="standard"> Please describe the situation in a few words: </label>
-                        <input type="checkbox" name="AxSituation" id="AxSituation"/>
+                        <input type="text" name="AxSituation" id="AxSituation"/>
 
                     <p> Please imagine the situation as vividly as possible (i.e., so that it feels very real).
                     Try to imagine you are actually in the situation.
                     Play out the specific parts of the situation in your head.
                     Try to imagine using all the things you would see, hear, and smell if you were really in that situation.
-                    Think about what thoughts or feelings you would experience if you were actually there. <p/>
+                    Think about what thoughts or feelings you would experience if you were actually there. </p>
 
-                    <p> Please click the button below and play this situation in your head for the next 20 seconds. We will tell you when the 20 seconds are up. <p/>
+                    <p> Please click the button below and play this situation in your head for the next 20 seconds. We will tell you when the 20 seconds are up. </p>
 
                     <button onclick="setTimeout(myFunction, 20000);"> Begin </button>
```

Comment:
On the original R34 study we used Javascript to make this single form feel like multiple pages.  Perhaps we need to do this again here for it to submit correctly.